evm/src/executor/abi.rs

```diff
@@ -680,6 +680,266 @@ pub static HARDHAT_CONSOLE_SELECTOR_PATCHES: Lazy<HashMap<Selector, Selector>> =
     ])
 });
 
+// Similar to hardhat's but ignores all specifiers except %s. %s works on all solidity values.
+// Support for other specifiers may be added in the future
+fn format_hardhat_log(spec: &str, v: &[&str]) -> String {
+    let mut result = spec.to_string().replace("%%", "%");
```

Comment:
what's the reasoning behind this?

I fear this would make it impossible to format things like %10 `"%%s", 10`

Reply:
Good catch. `%%s", 10` should be formatted as `%s 10`, which doesn't work with the current implementation.

Reply:
fixed.